doc/mgr/orchestrator.rst

```diff
@@ -285,7 +264,46 @@ Start/stop/reload::
     ceph orch service {stop,start,reload} <type> <name>
 
     ceph orch daemon {start,stop,reload} <type> <daemon-id>
+    
+
+.. _orchestrator-cli-placement-spec:
+    
+Placement Specification
+=======================
+
+In order to allow the orchestrator to deploy a *service*, it needs to
+know how many and where it should deploy *daemons*. The orchestrator 
+defines a placement specification:
+
+* To deploy three *daemons*, simply specify the count: ``3``
+* To deploy *daemons* on specific hosts, specify the host names: ``host1 host2 host3``
+* To deploy *daemons* on a subset of hosts, also specify the count: ``2 host1 host2 host3``
+
+    
+Configuring the Orchestrator CLI
+================================
+
+To enable the orchestrator, select the orchestrator module to use
+with the ``set backend`` command::
+
+    ceph orch set backend <module>
+
+For example, to enable the Rook orchestrator module and use it with the CLI::
 
+    ceph mgr module enable rook
+    ceph orch set backend rook
```

Comment:
We might as well show how to enable cephadm here too, since there are only 2 backends?

Reply:
I hope no user ever will need this.